packages/app/features/NetworkLayout/WalletCard/Address.tsx

```diff
@@ -31,6 +37,8 @@ export const WalletAddress: FC<Props> = ({ index, skin, token }) => {
 
 	const onCopy = () => {
```

Comment:
make a Valtio action for this, `navigator.clipboard.writeText` could become more complicated (because permission) to handle, put it under unified place reduce fragment.